service/service.go

```diff
@@ -525,6 +530,15 @@ func (s *service) getCSIVolume(vol *siotypes.Volume, systemID string) *csi.Volum
 	return vi
 }
 
+// getArryaInstallationID returns installation ID for the given system ID
+func (s *service) getArrayInstallationID(systemID string) (string, error) {
+	system, err := s.adminClients[systemID].FindSystem(systemID, "", "")
+	if err != nil {
+		return "", err
```

Comment:
Is there a unit test that tests the negative case in this scenario?

Reply:
Added UTs for the new method in latest commit.

Reply:
looks good, thanks!